packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -259,21 +264,13 @@ function getFluentUIApplePalette(): FluentUIApplePalette {
       light: '#2525253F',
       dark: '#5656567F',
     }),
-    neutralInverted: '#FFFFFF',
-    neutralForeground2: '#000000D8',
-    neutralBackground2: DynamicColorMacOS({
-      light: '#FFFFFF',
-      dark: '#555555',
-    }),
-    neutralStroke2: '#00000026',
-    neutralForeground3: DynamicColorMacOS({
-      light: '#272727',
-      dark: '#FFFFFF',
-    }),
-    neutralBackground3: DynamicColorMacOS({
-      light: '#0000000C',
-      dark: '#555555',
-    }),
+    neutralBackgroundInverted: macOSAliasColorTokens.neutralBackgroundInverted,
```

Comment:
It's kind of redundant to redefine the colors here, but let's keep it for now until we have all the colors updated?

Reply:
There will probably be lots of redundancy till we're fully integrated. Good to keep track where they are so we know to remove them, especially for the sake of bundle size.

Reply:
I think if we do move to fully design token based, then we would get rid of the apple theme altogether, and just have it be part of the "Fluent Theme" or "Default Theme", since we don't have native code for this theme anyway.